tools/flub/.eslintrc.cjs

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+    plugins: ["@typescript-eslint"],
+    extends: [
+        "oclif",
+        "oclif-typescript",
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended",
+        // require.resolve("@fluidframework/eslint-config-fluid"),
```

Comment:
Is the intention to enable this now, or later?

Reply:
Fixed.